packages/forma-36-react-components/src/components/Icon/svg/ArrowDown.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+
+function SvgArrowDown(props: React.SVGProps<SVGSVGElement>) {
+  return (
+    <svg width="1em" height="1em" viewBox="0 0 24 24" {...props}>
+      <path d="M7 10l5 5 5-5z" />
+      <path d="M0 0h24v24H0z" fill="none" />
+    </svg>
+  );
+}
+
+export default SvgArrowDown;
```

Comment:
@burakukula look!
This is basically what we need for an icon package

Reply:
yeah! exactly! 🙂